FAQ: add one TiKV question (#922)
pingcap/docs: FAQ.md

This is because the disk space is not enough. You need to add nodes or enlarge the disk space.

#### Why does the OOM (Out of Memory) error occur frequently in TiKV?

> TiKV uses the memory mainly for the block-cache of RocksDB, which is 40% of the system memory size by default. When the OOM error occurs frequently in TiKV, you should check if the value of `block-cache-size` is set too high. In addition, when multiple TiKV instances are deployed on a single machine, you need to explicitly configure the parameter to prevent multiple instances from using too much system memory that results in the OOM error. 

Reviewer: ```suggestion
The memory usage of TiKV mainly comes from the block-cache of RocksDB, which is 40% of the system memory size by default. When the OOM error occurs frequently in TiKV, you should check whether the value of `block-cache-size` is set too high. In addition, when multiple TiKV instances are deployed on a single machine, you need to explicitly configure the parameter to prevent multiple instances from using too much system memory that results in the OOM error. 
```

Author: Updated. PTAL again.
